hnn_core/network.py

```diff
@@ -400,6 +406,61 @@ def __repr__(self):
                  len(self.pos_dict['L5_basket'])))
         return '<%s | %s>' % (class_name, s)
 
+    def set_cell_positions(self, *, n_pyr_x=10, n_pyr_y=10,
+                           inplane_distance=1., layer_separation=1307.4):
+        """Set relative positions of cells arranged in a square grid
+
+        All ``hnn_core``-models are organised in a square grid of
+        ``n_pyr_x X n_pyr_y` pyramidal cells. Basket cells are included at a
```

Comment:
missing one backtick. In general, I suggest being conservative with fancy formatting because many users tend to check the docstrings on ipython where it is not rendered in HTML